extensions/amp-story-360/0.1/amp-story-360.md

```diff
@@ -0,0 +1,224 @@
+
+### Media Requirements
+
+The `amp-story-360` component renders videos and images in [equirectangular projection](https://en.wikipedia.org/wiki/Equirectangular_projection).
+This is the default format generated by many 360 cameras and 3D rendering software.
+
+<amp-img src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/SeanDoran-Quela-sol1462-edited_ver2-sm_600-300.jpg" layout="intrinsic" width="400" height="230">
+
+Credit: [NASA / JPL / MSSS / Seán Doran](https://informal.jpl.nasa.gov/museum/360-video).
+
+### Dimensions
+
+Use a 2:1 aspect ratio for the best distribution of resolution.
+
+### Size
+
+2048 x 1024 (2k) is recommended for images and video.
+
+### Configuration
+
+The `amp-story-360` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories) or [amp-video](https://amp.dev/documentation/components/amp-video/?format=stories).
+
+#### amp-image
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-img layout="fill" src="assets/image360.jpg"></amp-img>
+</amp-story-360>
+[/sourcecode]
+
+#### amp-video
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-video layout="fill" poster="assets/poster.jpg" autoplay loop>
+
+<source src="assets/video360.mp4" type="video/mp4" />
+</amp-video>
+</amp-story-360>
+[/sourcecode]
+
+Be sure to import the `amp-video` component.
+
+#### Defining a initial point of view (PoV)
+
+[sourcecode:html]
+<amp-story-360 
+   layout="fill"
+   heading-start="95" pitch-start="-10">
+...
+</amp-story-360>
+[/sourcecode]
+
+#### PoV to PoV animation
+
+[sourcecode:html]
+<amp-story-360 
+   layout="fill"
+   heading-start="95" pitch-start="-10"
+   heading-end="-45" pitch-end="-20"
+   duration="3s">
+...
+</amp-story-360>
+[/sourcecode]
+
+The `duration` attribute is required for animation.
+
+#### Gyroscope
+
+Use the device gyroscope sensor to set the image's orientation. The component defaults to the `heading-end` value.
+
+Serve the story over `https` to use this feature.
+
+[sourcecode:html]
+<amp-story-360 
+   layout="fill"
+   controls="gyroscope"
+   heading-start="95" pitch-start="-10"
+   heading-end="-45" pitch-end="-20"
+   duration="3s">
+...
+</amp-story-360>
+[/sourcecode]
+
+##### Activate 360 button
+
+If the device requires user permission to access the device’s gyroscope sensor an “activate 360” button will display.
+
+<amp-img src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/activate-360.png" layout="intrinsic" width="88.5" height="36">
+
+##### Animation fallback
+
+If the device does not support gyroscope or the user declines permission, the component will fallback to animating.
+
+## Attributes
+
+### Animation
+
+#### heading-start {float} optional
+
+Specifies the heading (horizontal angle in degrees) at the beginning of an animation.
```

Comment:
nit: centered on 0 (default), with a negative angle looking towards the left, positive angle to the right